src/x509_str.c

```diff
@@ -736,6 +736,10 @@ WOLFSSL_X509_STORE* wolfSSL_X509_STORE_new(void)
 #endif
 
 #if defined(OPENSSL_EXTRA) || defined(WOLFSSL_WPAS_SMALL)
+
+    /* Link store's new Certificate Manager to self by default */
+    store->cm->x509_store_p = store;
```

Comment:
Should we adopt the user's reproducing test case as a regression test somewhere in api.c?

Reply:
Added new test and supporting cert.